index.html

```diff
@@ -255,6 +255,62 @@
 
 ---
 
+# Continuous Analysis
+
+The Status Quo Doesn't Work:
+.task[
+- 66% of microarray gene expression experiments could not be reproduced without discrepancy (Ioannidis et al. 2009)
+
+- 50% of microarray gene experssion experiments can not be reproduced because of version number alone (Beaulieu-Jones & Greene 2016)
+
+]
+
+Even in the good cases:
+
+.task[
+- 280 hours for non-expert to reproduce drug-target network (Garijo et al. 2013) 
+]
+
+---
+
+# Continuous Analysis
```

Comment:
I'm also missing the lead in of how this relates to the overall objectives? I think you might be implicitly saying - you need to capture these things, but that's hard to do. Here is a way to automate it if you wish.